File: hfsfolder-vbnet/model/FolderOrder.vb
'
' * Created by SharpDevelop.
' * User: henrique.souza
' * Date: 08/04/2019
' * Time: 13:52
' * 
' * To change this template use Tools | Options | Coding | Edit Standard Headers.
' 

Namespace HfsFolder.Model
	''' <summary>
	''' Description of FolderOrder.
	''' </summary>
	Public Class FolderOrder
		
		Private codFolder As Integer

		Private order As Integer

		Public Sub New(codFolder As Integer, order As Integer)
			Me.codFolder = codFolder
			Me.order = order
		End Sub

		Public Sub New(folderOrder1 As FolderOrder)
			Me.codFolder = folderOrder1.getCodFolder()
			Me.order = folderOrder1.getOrder()
		End Sub

		Public Function getCodFolder() As Integer
			Return codFolder
		End Function

		Public Sub setCodFolder(codFolder As Integer)
			Me.codFolder = codFolder
		End Sub

		Public Function getOrder() As Integer
			Return order
		End Function

		Public Sub setOrder(order As Integer)
			Me.order = order
		End Sub

	End Class
End Namespace

File: hfsfolder-python/model/FolderOrder.py
# 
# * Created by SharpDevelop.
# * User: henrique.souza
# * Date: 08/04/2019
# * Time: 13:52
# *
# * To change this template use Tools | Options | Coding | Edit Standard Headers.
# 
class FolderOrder:
	""" <summary>
	 Description of FolderOrder.
	 </summary>
	"""
	def __init__(self, codFolder, order):
		self._codFolder = codFolder
		self._order = order

	def getCodFolder(self):
		return self._codFolder

	def setCodFolder(self, codFolder):
		self._codFolder = codFolder

	def getOrder(self):
		return self._order

	def setOrder(self, order):
		self._order = order